tests/test_distribmat/test_distribmat.py

```diff
@@ -82,49 +69,39 @@ def _build_inputs(self):
         sets_file.batches = 5
         sets_file.inactive = 0
         sets_file.particles = 1000
-        sets_file.source = Source(space=Box([-1, -1, -1], [1, 1, 1]))
-        sets_file.output = {'summary': True}
+        sets_file.source = openmc.Source(space=openmc.stats.Box(
+            [-1, -1, -1], [1, 1, 1]))
         sets_file.export_to_xml()
 
-
         ####################
         # Plots
         ####################
 
-        plots_file = openmc.Plots()
-
-        plot = openmc.Plot(plot_id=1)
-        plot.basis = 'xy'
-        plot.color_by = 'cell'
-        plot.filename = 'cellplot'
-        plot.origin = (0, 0, 0)
-        plot.width = (7, 7)
-        plot.pixels = (400, 400)
-        plots_file.add_plot(plot)
-
-        plot = openmc.Plot(plot_id=2)
-        plot.basis = 'xy'
-        plot.color_by = 'material'
-        plot.filename = 'matplot'
-        plot.origin = (0, 0, 0)
-        plot.width = (7, 7)
-        plot.pixels = (400, 400)
-        plots_file.add_plot(plot)
-
-        plots_file.export_to_xml()
+        plot1 = openmc.Plot(plot_id=1)
```

Comment:
I think in this case, there were plots so that one could make sure distributed materials were really working (by visually inspecting)